src/isIsogram.test.js

```diff
@@ -6,4 +6,16 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('Should return false for "look"', () => {
```

Comment:
For this case I would use description like this:
'should return false if entered string is lowercase and has repeatings'
Think of other test descriptions.